sdk/core/System.Net.ClientModel/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
```

Comment:
If this is the first PR to merge for ClientModel, we should update this to reflect the types it contains and the actual release history.

Also, if we release a beta package from this branch, we will need to update this prior to running the release pipeline.